_posts/2024-05-02-vega-tsvb-mds-visualizations.markdown

```diff
@@ -0,0 +1,215 @@
+---
+layout: post
+title:  "Creating visualizations using multiple data sources"
+authors:
+ - wronghuy
+date: 2024-05-28
+categories:
+ - technical-post
+meta_keywords: multiple data source, vega, tsvb
+meta_description: Learn how to create TSVB and Vega visualizations with the multiple data sources feature.
+excerpt: OpenSearch versions 2.13 and 2.14 now support multiple data sources for Vega and TSVB visualization types. This blog post shows you how to use this feature.
+---
+
+# Visualizing data from multiple data sources with TSVB and Vega
+
+## Introduction
+
+[Multiple data sources](https://opensearch.org/blog/multiple-data-source/) gives users the capability to visualize data from various OpenSearch clusters. To date, only certain visualization types have been available. With the release of OpenSearch Dashboards 2.13 and 2.14, multiple data sources is now compatible with [Vega](https://opensearch.org/docs/latest/dashboards/visualize/vega/) and [Time-Series Visual Builder (TSVB)](https://opensearch.org/docs/latest/dashboards/visualize/viz-index/#tsvb) visualizations types.
+
+Read on to learn how you can visualize your data from multiple data sources using these visualization types with OpenSearch Dashboards.
+
+## What are TSVB and Vega?
+
+TSVB is a data visualization tool for creating detailed time-series visualizations. Its key feature is the ability to add annotations or markers at specific time points based on index data, enabling associations between multiple indexes. In addition to time-series visualizations, TSVB offers area, line, metric, gauge, markdown, and data table panels. As of OpenSearch Dashboards version 2.14, users can choose the data source for their visualizations.
```

Comment:
```suggestion
TSVB is a data visualization tool for creating detailed time-series visualizations. Its key feature is the ability to add annotations or markers at specific time points based on index data, enabling associations between multiple indexes. In addition to time-series visualizations, TSVB offers area, line, metric, gauge, markdown, and data table panels. As of OpenSearch Dashboards version 2.14, users can choose the data source(s) for their visualizations.
```

Reply:
Users can choose only one data source at a time for their TSVB visualizations This edit implies that TSVB can support multiple concurrent data sources

Reply:
Ah, OK, I was thinking along the lines of "multiple." Go ahead and ignore 😄